cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -117,7 +117,11 @@ runNode loggingLayer npm@NodeCLI{protocolFiles} = do
         Left err -> (putTextLn $ renderProtocolInstantiationError err) >> exitFailure
         Right (SomeConsensusProtocol p) -> pure $ SomeConsensusProtocol p
 
-    tracers <- mkTracers (ncTraceConfig nc) trace
+    txsProcessedCounter :: MVar Integer <- newMVar 0
```

Comment:
in the near future, this can become a data structure that contains more state variables.

Reply:
Exactly.